### DISCUSSION TITLE
[bitnami/thanos] Query multiple object storage buckets from Thanos instance

### DISCUSSION THREAD

**Discussion by MysterioO** [thanos-io/thanos#6659] [Questions & Answers]
Our current set-up is that we have to have multiple metric buckets to separate certain cluster data. We are wanting to switch to Thanos bitnami helm chart. Currently we are having individual thanos-store pods for each bucket, however on first glance this is not possible with Thanos helm chart. We are considering maybe deploying a sidecar for each object storage in storegateway statefulSet, but interested if others are using different/better approach.

**Comment by ylz** [comment]
@MysterioO seems to be a fairly old question, but did you find any way to use multiple buckets with Bitnami Thanos Helm charts?
